nilearn/tests/test_signal.py

```diff
@@ -279,7 +279,7 @@ def test_clean_confounds():
     # With signal: output must be orthogonal to confounds
     cleaned_signals = nisignal.clean(signals + noises, confounds=confounds,
                                       detrend=True, standardize=False)
-    assert_true(abs(np.dot(confounds.T, cleaned_signals)).max() < 100. * eps)
+    assert_true(abs(np.dot(confounds.T, cleaned_signals)).max() < 1000. * eps)
```

Comment:
I feel this has been discussed but I couldn't find it anywhere. How come we need to relax this condition, i.e. somehow that the cleaned_signals and confounds are less orthogonal ?